src/libopensc/card-sc-hsm.c

```diff
@@ -1279,9 +1287,7 @@ static int sc_hsm_initialize(sc_card_t *card, sc_cardctl_sc_hsm_init_param_t *pa
 		*p++ = params->bio2.len;
 		memcpy(p, params->bio2.value, params->bio2.len);
 		p += params->bio2.len;
-	}
-
-	sc_format_apdu(card, &apdu, SC_APDU_CASE_3_SHORT, 0x50, 0x00, 0x00);
+  } sc_format_apdu(card, &apdu, SC_APDU_CASE_3_SHORT, 0x50, 0x00, 0x00);
```

Comment:
Wrong indentation of closing bracket and the `sc_format_apdu()` should start on the new line. Generally, this diff does not change anything useful.

Reply:
Sorry, this slipped in by accident.